db-modifications.sql

```diff
@@ -0,0 +1,20 @@
+/* Complex schemas are not great in Sails auto migrations
+If you are running a CSMM instance with many servers or have high data throughput these commands can help you out. */
+
+--- Indexes
+ALTER TABLE sdtdconfig ADD INDEX IDX_sdtdconfig_server (`server` ASC);
```

Comment:
I think this also should be unique, there's no point in having the same config multiple times